Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Image/StreamingImageAsset.h

```diff
@@ -127,6 +130,9 @@ namespace AZ
             uint32_t m_totalImageDataSize = 0;
 
             StreamingImageFlags m_flags = StreamingImageFlags::None;
+
+            // Cached value of the average color of this image (alpha-weighted average in case of 4-component images)
+            AZ::Color m_meanColor;
```

Comment:
I'm not sure about adding that mean color property here given its limitations and very niche use case. If understand correctly it would also incur a penalty for all texture, regardless of their usage, to decompress, linearize, and perform the mean estimation (granted only at import time). Have you profiled it for 8k textures DXT5 compressed?
Have you considered instead to use the bottom mip information? Depending on how it is calculated it could be very similar.

Reply:
On a second look, it is way more natural (and obvious in hindsight) to compute the average color directly within the ImageConvert process (it should come nearly for free during/after the mip map computation step). I'll go down that route!